lib/structures/GuildChannel.js

```diff
@@ -13,7 +14,7 @@ const PermissionOverwrite = require("./PermissionOverwrite");
 * @prop {String} id The ID of the channel
 * @prop {String} name The name of the channel
 * @prop {Boolean} nsfw Whether the channel is an NSFW channel or not
-* @prop {String?} parentID The ID of the category this channel belongs to
+* @prop {String?} parentID The ID of the category this channel belongs to or the message ID where the thread originated from (thread channels only)
```

Comment:
```suggestion
* @prop {String?} parentID The ID of the category this channel belongs to or the channel ID where the thread originated from (thread channels only)
```
right?

Reply:
No

Reply:
https://discord.com/developers/docs/resources/channel#channel-object
Then why does it say it here?
and in code you also use `.channels.get(.parentID)` ?

Reply:
[docs](https://discord.com/developers/docs/resources/channel#channel-object-channel-structure) say parent_id is the id of the channel the thread was made on
>  for threads: id of the text channel this thread was created